src/components/OutlinedCardList.tsx

```diff
@@ -13,16 +13,19 @@ interface CardProps {
   primary: boolean;
   rightIcon?: JSX.Element;
   columns?: boolean;
+  grayBackground?: boolean;
```

Comment:
If I'm understanding this correctly, it seems like grey background represents a 'read only' state, so I think it would make more sense to use a prop like `readOnly` that represents its actual use case, not just the actual color.

Reply:
see comment above ^ if there's a distinction between gray background & text i think i want to keep the distinction in the naming